samples/compiler_plugins/xnnpack/test/fully-connected-e2e.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: iree-compile --iree-hal-target-backends=llvm-cpu --iree-plugin=xnnpack %s | \
+// RUN: iree-run-module \
+// RUN:     --device=local-sync \
+// RUN:     --executable_plugin=$IREE_BINARY_DIR/samples/custom_dispatch/xnnpack/plugin/system_plugin$IREE_DYLIB_EXT \
+// RUN:     --module=- \
+// RUN:     --function=main \
+// RUN:     --input=1x2x8xi8=1 \
+// RUN:     --input=4x8xi8=2 \
+// RUN:     --input=4x8xi8=8 | \
+// llvm-lit does not like checking for '[['
```

Comment:
You can escape like this: `{{\[}}[0], [1, 2]]`.

Reply:
Nice!